GEOSwift.xcodeproj/project.pbxproj

```diff
@@ -522,6 +558,7 @@
 			buildSettings = {
 				CLANG_CXX_LIBRARY = "compiler-default";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				COMPRESS_PNG_FILES = NO;
```

Comment:
I disabled these PNG compression build settings on the test target so that I could do a byte-for-byte comparison between a new render and a saved image.